code/datums/keybinding/xeno.dm

```diff
@@ -65,343 +65,343 @@
 /datum/keybinding/xeno/transfer_plasma
 	key = "Unbound"
 	name = "transfer_plasma"
-	full_name = "Transfer Plasma"
+	full_name = "Drone: Transfer Plasma"
 	description = "Give some of your plasma to a teammate."
 	keybind_signal = COMSIG_XENOABILITY_TRANSFER_PLASMA
 
 /datum/keybinding/xeno/larval_growth_sting
 	key = "Unbound"
 	name = "larval_growth_sting"
-	full_name = "Larval Growth Sting"
+	full_name = "Drone: Larval Growth Sting"
```

Comment:
This path works for both this ability and `Defile` as well, no?

Reply:
Ah yeah it might, in which case i'll rename to xeno